src/filter-pattern/pattern-time-relative.ts

```diff
@@ -82,6 +84,8 @@ export const TIME_RELATIVE_PATTERN_DEFINITION: FilterPatternDefinition<TimeRelat
       const rangeStep = -timeShift.getArgAsNumber(2)!;
       if (!(rangeStep > 0)) return;
 
+      const timezone = timeShift.getArgAsString(3);
```

Comment:
Please add checks to make sure that, if a timezone is used it is the same in all the instances of the pattern